Python/executor_cases.c.h

```diff
@@ -183,6 +182,7 @@
         }
 
         case TO_BOOL_ALWAYS_TRUE: {
+            static_assert(INLINE_CACHE_ENTRIES_TO_BOOL == 3, "incorrect cache size");
```

Comment:
This still isn't resolved?

Reply:
No, I think this might be related to the change in `Lib/_opcode_metadata.py`as well; the op from the family name isn't getting inserted into `instrs`

I'm having some trouble getting the name into `instrs` / parsed as an instruction. Any advice?